schemas/base.yml

```diff
@@ -14,8 +14,11 @@
       description: >
         Date/time when the event originated.
 
-        For log events this is the date/time when the event was generated, and
-        not when it was read.
+        This is the date/time extracted from the event, representing when the
+        event was generated by the source.
+
+        If the event source has no original timestamp, this value must be
```

Comment:
"this value must be" seems too restrictive.  We've previously defined `must` and `should` semantics to indicate whether ECS will break or will cause unexpected results.   I suggest changing this to "this value is typically"